src/unittests/parse_imports.spec.ts

```diff
@@ -0,0 +1,64 @@
+export const description = `
+Test for "parseImports" utility.
+`;
+
+import { makeTestGroup } from '../common/framework/test_group.js';
+import { parseImports } from '../common/util/parse_imports.js';
+
+import { UnitTest } from './unit_test.js';
+
+class F extends UnitTest {
+  test(content: string, expect: string[]): void {
+    const got = parseImports('a/b/c.js', content);
+    const expectJoined = expect.join('\n');
+    const gotJoined = got.join('\n');
+    this.expect(
+      expectJoined === gotJoined,
+      `
+expected: ${expectJoined}
+got:      ${gotJoined}`
+    );
+  }
+}
+
+export const g = makeTestGroup(F);
+
+g.test('empty').fn(t => {
+  t.test(``, []);
+  t.test(`\n`, []);
+  t.test(`\n\n`, []);
+});
+
+g.test('simple').fn(t => {
+  t.test(`import 'x/y/z.js';`, ['a/b/x/y/z.js']);
+  t.test(`import * as blah from 'x/y/z.js';`, ['a/b/x/y/z.js']);
+  t.test(`import { blah } from 'x/y/z.js';`, ['a/b/x/y/z.js']);
+});
+
+g.test('multiline').fn(t => {
```

Comment:
Probably should also test multiple imports in the same file

Reply:
Done